packages/core/src/Stores/Helpers/client-notifications.js

```diff
@@ -462,8 +462,11 @@ const checkAccountStatus = (
         needs_verification.includes('identity') &&
         (identity?.status !== 'expired' || identity?.status !== 'pending');
 
-    if (needs_poa && !(document.status === 'expired')) addNotificationMessage(clientNotifications().needs_poa);
-    if (needs_poi && !(identity.status === 'expired')) addNotificationMessage(clientNotifications().needs_poi);
+    const should_request_poa = needs_poa && !(document.status === 'expired');
```

Comment:
any reason why are we checking expiry twice, didnt we check it on line 458?. also `document.status !== 'expired'`